### PR TITLE
[idpo] Added idpo ontology

### PR DESCRIPTION
**Pull request by philhag**

### CODE REVIEW

File: idpo/.htaccess [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# In case of accept header <application/rdf+xml>
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^$ https://icdd.vm.rub.de/ontology/idpo/idpo.rdf [R=302,NE,L]
+
+# In case of accept header <application/ld+json>
+RewriteCond %{HTTP_ACCEPT} application/ld+json
+RewriteRule ^$ https://icdd.vm.rub.de/ontology/idpo/idpo.jsonld [R=302,NE,L]
+
+# If suffix ttl, redirect to turtle version
+RewriteRule ^idpo.ttl$ https://icdd.vm.rub.de/ontology/idpo/idpo.ttl [R=302,NE,L]
```
Comment (dgarijo) [inline idpo/.htaccess:30]: This will be https://w3id.org/idpo/idpo.ttl Is this what you intend?

**Comment by dgarijo** [comment]
Please see my comment above. Also, 303 redirect is recommended for semantic resources instead of 302 (I see you used it only in the turtle)

File: idpo/.htaccess (hunk 1 above)
Comment (philhag) [reply idpo/.htaccess:30]: Right, its intented to serve the turtle file either by requesting using the <text/turtle> header on the base path or specifying the ttl file directly, e.g for weblinks to that file

**Comment by philhag** [comment]
I changed response code to 303 for ontology resources.

**Pull request merged**